umbral/umbral.py

```diff
@@ -6,17 +6,22 @@
 from umbral.point import Point
 from umbral.keys import UmbralPrivateKey, UmbralPublicKey
 from umbral.dem import UmbralDEM
-from umbral.utils import poly_eval, lambda_coeff, hash_to_bn, kdf
+from umbral.utils import poly_eval, lambda_coeff, hash_to_bn, kdf, hash_to_point
 
 
 class UmbralParameters(object):
     def __init__(self):
         self.curve = ec.SECP256K1()
         self.g = Point.get_generator_from_curve(self.curve)
         self.order = Point.get_order_from_curve(self.curve)
-        self.h = self.g
-        self.u = self.g
+        
+        g_bytes = self.g.to_bytes(is_compressed=True)
+
+        constant_u = b'NuCypherKMS/UmbralParameters/u'
```

Comment:
Any interest in naming this something like, "domain"?  

`constant_u` doesn't really explain *why* this is here.

Reply:
It makes sense to me, but it could be renamed to `constant_u_seed` or something similar. `domain` doesn't make much sense to me.

Reply:
I don't love that either.  It's not any more "constant" than any other bytes argument to a function.

As for the "domain" idea: this is the seed (that part is good) that places this `UmbralParameters` as within the same domain as all other instances with like parameters.  So, it really is a domain denominator.

`domain_seed`?

I also don't see a good reason to have the user set both values; just have them give one and we'll append to it in order to make two.

Reply:
So you mean we use `domain_seed + b'h'` and `domain_seed + b'u'`, right?

Reply:
Yeah, if `b'h'` and `b'u'` are the distinguishing tokens you want to use.

Reply:
Sounds good.